SEDaily-IOS/HeaderView.swift

```diff
@@ -199,40 +206,38 @@ extension HeaderView {
     
     func addScore(active: Bool) {
         if active == false {
-            self.scoreLabel.text = String(model.score!)
-            self.model.update(isUpvoted: false)
+            self.scoreLabel.text = String(model.score)
+            self.model.isUpvoted = false
             upVoteButton.isSelected = self.model.isUpvoted
             downVoteButton.isSelected = self.model.isDownvoted
             return
         }
 
-        guard let score = self.model.score else { return }
-        guard var int = Int(score) else { return }
+        var int = self.model.score
```

Comment:
You can combine the following three lines into one, e.g. self.scoreLabel.text = String(self.model.score + 1)

Reply:
👍